ci/check-params-env.sh

```
@@ -770,6 +818,18 @@ process_file() {
             continue
         }
 
+        if [[ "${IMAGE_URL}" == "dummy" ]]; then
+            if [[ "${1}" == "${PARAMS_LATEST_ENV_PATH}" ]]; then
```

Shellcheck (0.11.0)

[info] Make sure not to read and write the same file in the same pipeline.

(SC2094)